routes/api/upgrade/prechecks.js

```diff
@@ -9,10 +9,11 @@ router.get('/', function(req, res) {
         res.status(500).json({'errors': errors});
     } else {
         res.status(200).json({
```

Comment:
This is meant to simplify testing this. Failed checks are not errors in this context.